serverless.yaml

```diff
@@ -569,6 +612,12 @@ resources:
           KmsKeyId: !GetAtt LogKMSKey.Arn
         DependsOn:
           - LogKMSKey
+      ApiGatewayMethodMetadataGet:
+        Metadata:
+          cfn_nag:
+            rules_to_suppress:
+              - id: W45
+                reason: 'This API endpoint should not have authentication required (due to the FHIR spec)'

```

Comment:
rephrase: `should not require authentication`

Reply:
```suggestion
                reason: 'This API endpoint should not require authentication (due to the FHIR spec)'

```